src/javascript/app/pages/trade/price.js

```diff
@@ -246,22 +246,22 @@ const Price = (() => {
             CommonFunctions.elementInnerHtml(contract_multiplier, data.multiplier ? formatMoney(currentCurrency, data.multiplier, false, 0, 2) : '-');
 
             dataManager.setPurchase({
-                [`${position}Amount`]      : data.display_value ? formatMoney(currentCurrency, data.display_value, true) : '-',
-                [`${position}PayoutAmount`]: data.payout ? formatMoney(currentCurrency, data.payout, true) : '-',
-                [`${position}Multiplier`]  : data.multiplier ? formatMoney(currentCurrency, data.multiplier, true, 0, 2) : '-',
-                currency                   : getCurrencyDisplayCode(currentCurrency),
+                [`${position}_amount`]       : data.display_value ? formatMoney(currentCurrency, data.display_value, true) : '-',
+                [`${position}_payout_amount`]: data.payout ? formatMoney(currentCurrency, data.payout, true) : '-',
+                [`${position}_multiplier`]   : data.multiplier ? formatMoney(currentCurrency, data.multiplier, true, 0, 2) : '-',
+                currency                     : getCurrencyDisplayCode(currentCurrency),
             });
 
             if (data.longcode && window.innerWidth > 500) {
                 dataManager.setPurchase({
-                    [`${position}Description`]: data.longcode,
+                    [`${position}_description`]: data.longcode,
                 });
 
                 if (description) description.setAttribute('data-balloon', data.longcode);
                 if (longcode) CommonFunctions.elementTextContent(longcode, data.longcode);
             } else {
                 dataManager.setPurchase({
```

Comment:
please also rename dataManager variable to data_manager in the file where it's exported from🙏🏻

Reply:
It's a class shouldn't we keep it as camelCase?

Reply:
should be fine in snake_case since it's a class instance (an object) :)